featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -788,12 +790,33 @@ def gen_feat_names(self):
             assert base_feature.unique_name() != join_time_split.unique_name()
 
 
-def test_tranform_stack_agg(es):
-    topn = ft.Feature(es['log']['product_id'],
-                      parent_entity=es['customers'],
-                      primitive=NMostCommon(n=3))
-    with pytest.raises(AssertionError):
-        ft.Feature(topn, primitive=Percentile)
+def test_stacking_of_multi_output_transform_feat(es):
```

Comment:
Instead of adding new test case I think we can update the "test_make_transform_multiple_output_features" test to check for stacking